### ISSUE TITLE
scan results are not displayed

### ISSUE THREAD

**Issue by FGAGwendalG** [mozilla/http-observatory-website#279]
Hello,

The scan results for the following site are not displayed.

https://observatory.mozilla.org/analyze/vdi.figeac-aero.com

Kind regards,

**Comment by gene1wood** [comment]
Hmmm. so the lets encrypt cert at `www.figeac-aero.com` works but the wildcard cert from Sectigo at `vdi.figeac-aero.com` doesn't.

**Comment by FGAGwendalG** [comment]
Yes, it's a different site, that's why.

**Comment by FGAGwendalG** [comment]
Hello,

Do you have any news related to this issue?

Kind regards,

**Comment by gene1wood** [comment]
No, unfortunately I haven't had an opportunity to look into it.

**Comment by Ansh101112** [comment]
Hello,
As this is a different website.

Have you more information about this?

Kind, Regards

**Comment by Ansh101112** [comment]
Hello,
Yes, I got it now and will be contributing to other stuff too.
Thank you for reaching out to me.
